.travis.yml

```diff
@@ -1,16 +1,25 @@
 language: go
 go_import_path: github.com/csi-driver/csi-driver-smb
 dist: bionic
-
+services:
+  - docker
 go:
   - "1.13.9"
 
 before_install:
   - GO111MODULE=off go get github.com/mattn/goveralls
+  - mkdir /tmp/share
```

Comment:
can you make these 3 lines of scripts to be included in integration-test test scripts